inductiva/tasks/task.py

```diff
@@ -155,3 +156,22 @@ class _PathParams(TypedDict):
 
     def _get_path_params(self) -> _PathParams:
         return {"task_id": self.id}
+
+    def get_execution_time(self) -> float:
+        """Get the time the task took to complete.
+
+        Returns:
+            The time in seconds.
+        """
+
+        if self.get_status() != TaskStatusCode.SUCCESS:
+            raise RuntimeError("Task is not completed.")
+
+        else:
+            info = dict(self._api.get_task(self._get_path_params()).body)
+            end_time = datetime.strptime(str(info["end_time"]),
+                                         "%Y-%m-%dT%H:%M:%S.%f+00:00")
+            start_time = datetime.strptime(str(info["start_time"]),
+                                           "%Y-%m-%dT%H:%M:%S.%f+00:00")
```

Comment:
What is the type of `info["end_time"]`? Can you avoid the conversion to string?

Reply:
Ok, thank you for clarifying @david451lima. I was wondering if we could avoid the `str` casting, and instead instantiate the datetimes in some other way.